app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/audit/RegistrationRequestEvent.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.audit
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.address.Address
+
+final case class RegistrationRequestEvent(
+  prePopulatedUserData: Option[RegistrationPrePopulatedUserData],
+  manuallyEnteredData: Option[RegistrationManuallyEnteredData]
+)
+
+final case class RegistrationPrePopulatedUserData(
+  regime: String,
+  emailAddress: EmailAuditDetails
```

Comment:
this should be optional, as should the email address in `RegistrationManuallyEnteredData` - it's either one or the other. We don't want to be sending blank values for them in the event if they're not supposed to be there